dolphinscheduler-ui-next/src/views/security/user-manage/components/use-modal.tsx

```diff
@@ -0,0 +1,236 @@
+import { ref, watch, computed } from 'vue'
```

Comment:
If there is no file returned by dom, you do not need to use tsx, it is recommended to change it to ts.